docs/extending/adding_reports.md

```diff
@@ -178,6 +178,31 @@ def register_unpublished_changes_report_url():
 
 Here, we use the `AdminOnlyMenuItem` class to ensure our report icon is only shown to superusers. To make the report visible to all users, you could replace this with `MenuItem`.
 
+## Setting up permission restriction
+
+To make sure that the reports are only accessible by the `Admin` it is required to set up a permission restriction on the report view itself this can be done by adding a `dispatch` method to the existing `UnpublishedChangesReportView` view .
+
+```python
+from wagtail.admin.auth import permission_denied
+
+class UnpublishedChangesReportView(PageReportView):
+    header_icon = 'doc-empty-inverse'
```

Comment:
Since we're giving the full code immediately below, we don't really need to include all of the existing code here - everything from here up to the `dispatch` method can be skipped and replaced with a `# ...` comment to let them know the existing code has been omitted.

The `from wagtail.admin.auth import permission_denied` import also needs to be added to the final full code.